build/civicrm/setup-plugins/DockerMonolog.civi-setup.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * @file
+ *
+ * Install and configure Monolog to direct all logs to stdout
+ */
+
+if (!defined('CIVI_SETUP')) {
+  exit("Installation plugins must only be loaded by the installer.\n");
+}
+
+\Civi\Setup::dispatcher()
+  ->addListener('civi.setup.init', function (\Civi\Setup\Event\InitEvent $e) {
+    $e->getModel()->extensions[] = 'monolog';
+  });
+
+\Civi\Setup::dispatcher()
+  ->addListener('civi.setup.installDatabase', function (\Civi\Setup\Event\InstallDatabaseEvent $e) {
+    if (!in_array('monolog', $e->getModel()->extensions)) {
+      // in case installing Monolog extension was overridden by user
+      return;
+    }
+
+    \Civi\Setup::log()->info(sprintf('[%s] Handle %s', basename(__FILE__), 'installDatabase'));
```

Comment:
What happens when an attempt to log an info message occurs prior to initiation of logging infrastructure? Perhaps we should hold off until successful install has completed.

Reply:
This is standard boilerplate from all of the installer plugins, and is using a bespoke logger I think. It is printed to screen if you I stall with `cv`, not sure if it goes anywhere on web install.